### PR TITLE
fix prod build so tiles are rendered

### LINKED ISSUE #13: Production build does not display tiles
When running a production build, with `buildOptimizer` and `optimization` set to true, no tiles are displayed.
Tiles are properly fetched from blob storage, but the tiles are being interpreted as an unknown type and nothing is read from it

### PR DESCRIPTION
**Pull request by aruniverse**
fixes production build of app so tiles are rendered
clean up some warnings about cjs imports

### CODE REVIEW

**Comment by aruniverse** [comment]
partially dependent on https://github.com/iTwin/itwinjs-core/pull/4785

File: package.json [hunk 1]
```diff
@@ -48,6 +49,8 @@
     "@angular/compiler-cli": "~13.3.0",
     "@types/jasmine": "~3.10.0",
     "@types/node": "^12.11.1",
+    "cross-env": "7.0.3",
+    "http-server": "^14.1.1",
```
Comment (johnnyd710) [inline package.json:53]: you can remove these deps, I only added them to help with testing the issue (I had used them in the package.json scripts)

File: src/environments/environment.local.ts [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
+    redirectUri: "http://localhost:3000/",
+    postSignoutRedirectUri: "http://localhost:3000/",
     responseType: "code",
-    authority: "https://ims.bentley.com"
+    authority: "https://ims.bentley.com",
   },
-  iTwinId: "",
-  iModelId: "",
+  iTwinId: "7b1c22ff-ea4a-451d-b1fb-9d657262995a",
+  iModelId: "4a07c2bb-5e05-4c8c-b89c-8e7142ead5f0",
```
Comment (johnnyd710) [inline src/environments/environment.local.ts:20]: Did you mean to commit a specific itwin/imodel?

File: angular.json [hunk 3]
```diff
@@ -71,8 +72,12 @@
...
             },
             "development": {
-              "buildOptimizer": false,
-              "optimization": false,
+              "verbose": true,
+              "aot": true,
+              "statsJson": true,
+              "showCircularDependencies": true,
+              "buildOptimizer": true,
+              "optimization": true,
```
Comment (johnnyd710) [inline angular.json:80]: in development you might still want these as `false` for easier debugging

File: package.json (hunk 1 above)
Comment (aruniverse) [reply package.json:53]: i thought it was helpful for testing prod builds locally so i kept it in

File: src/environments/environment.local.ts (hunk 2 above)
Comment (aruniverse) [reply src/environments/environment.local.ts]: nope, beat me to commenting on it before i pushed the reverts up. resolved

File: angular.json (hunk 3 above)
Comment (aruniverse) [reply angular.json:80]: valid, but i dont like how dev and prod builds will differ (with these being enabled by default in prod) so i left them in. i can remove if you dont like that

Comment (johnnyd710) [reply angular.json:80]: Just seems weird to minify / tree shake / optimize in development. Maybe add a `"build:prod": ng build --prod` script to the package.json, that way it's clear how to run and check for a prod build. Feel free to resolve if you disagree

**Review by johnnyd710 — APPROVED** [review]

**Pull request merged**